tools/sunlight.go

```diff
@@ -125,7 +125,7 @@ func main() {
 	 insert into issuerReputation(
 		issuer,
 		issuerInMozillaDB,
-		validPeriodTooLongNormalizedScore, validPeriodTooLongRawscore,
+		validPeriodTooLongNormalizedScore, validPeriodTooLongRawScore,
```

Comment:
Does this also need to be fixed in the create table statement? (line 75, but github won't let me comment there)